agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/HttpServerTracer.java

```diff
@@ -16,35 +16,14 @@
 
```

Comment:
I moved the core logic from HttpServerTracer to HttpServerTracerBase. This new base class does not have a tracer field. I needed to create this base class to use my own tracer and not the default tracer initialized by HttpServerTracer. I kept all the logic involving Tracers in this abstract class. This approach limited the number of code changes I needed to make